### PR TITLE
History feature

### PR DESCRIPTION
**Pull request by Jay-Gadhiya**
Feature added:
- added into history, remove from history, clear history

### CODE REVIEW

**Review by gopalbbharadva — APPROVED** [review]
Functionality working fine , good to go.

File: src/Components/aside/aside.jsx [hunk 1]
```diff
@@ -10,15 +10,13 @@ import { Link, NavLink } from "react-router-dom";
 export const Aside = () => {
 
     const getActiveStyle = ({ isActive }) => ({
-        color: isActive ? "#fdca3e" : "#ffffff",
+        color: isActive ? "#4285F4" : "#ffffff",
```
Comment (kart-c) [inline src/Components/aside/aside.jsx:13]: you can toggle classes instead of inline styles

File: src/Utility-functions/likeHandler.js [hunk 2]
```diff
@@ -16,6 +16,7 @@ export const getLikeHandler = async (authState, dataStoreDispatch) => {
 }
 
 export const addToLike = async (video, authState, dataStoreDispatch, navigate) => {
+    console.log(authState.token);
```
Comment (kart-c) [inline src/Utility-functions/likeHandler.js]: you can remove `console.logs`

File: src/Utility-functions/likeHandler.js [hunk 3]
```diff
@@ -25,7 +26,7 @@ export const addToLike = async (video, authState, dataStoreDispatch, navigate) =
             }
             
         } catch (error) {
-            alert(error);
+            alert(error);6
```
Comment (kart-c) [inline src/Utility-functions/likeHandler.js]: I think this is a typo `6`

**Review by kart-c — APPROVED** [review]

**Pull request merged**